src/Evolution/Systems/Cce/Actions/InitializeWorldtubeBoundary.hpp

```diff
@@ -140,6 +142,51 @@ struct InitializeWorldtubeBoundary<H5WorldtubeBoundary<Metavariables>>
   using typename base_type::simple_tags_from_options;
 };
 
+/*!
+ * \ingroup ActionsGroup
+ * \brief Initializes a KleinGordonH5WorldtubeBoundary
+ *
+ * \details Uses:
+ * - initialization tag
+ * `Cce::Tags::H5WorldtubeBoundaryDataManager`,
+ * `Cce::Tags::KleinGordonH5WorldtubeBoundaryDataManager`.
```

Comment:
We don't have the `initialization_tag` type alias anymore, it's called `simple_tags_from_options` now. I know it's incorrect for the other ones as well. Could you please fix those dox in a separate commit?

Reply:
Done!